sagemaker_studio_image_build/cli.py

```diff
@@ -85,6 +107,18 @@ def main():
         "--bucket",
         help="The S3 bucket to use for sending data to CodeBuild (if None, use the SageMaker SDK default bucket).",
     )
+    build_parser.add_argument(
+        "--vpc-id",
+        help="The Id of the VPC that will host the CodeBuild Project (such as vpc-05c09f91d48831c8c).",
+    )
+    build_parser.add_argument(
+        "--subnets",
+        help="The comma-separated list of subnet ids for the CodeBuild Project (such as subnet-0b31f1863e9d31a67)",
+    )
+    build_parser.add_argument(
+        "--security-groups",
+        help="The comma-separated list of security group ids for the CodeBuild Project (such as sg-0ce4ec0d0414d2ddc).",
```

Comment:
In the VPC parameter, we're calling it `vpc-id` explicitly whereas in the subnets and security groups we're just calling them `subnets/security-groups` though we're expecting the user to pass IDs. I recommend we make the terminology here consistent

* --vpc, --subnets, --security-groups
OR
* --vpc-id, --subnet-ids, --security-groups-ids

My preference is the latter as it makes it clear to the user that the IDs are expected and not any other attribute of the VPC/Subnet/SecurityGroups like name

Reply:
Standardized the input arguments using the latter of the two options